### PR TITLE
feat: added rules page

### PR DESCRIPTION
**Pull request by muskan-gupta-0803**

### CODE REVIEW

**Comment by shivamSspirit** [comment]
looks good but you can also declare root variables in sperate CSS files

**Comment by muskan-gupta-0803** [comment]
> looks good but you can also declare root variables in sperate CSS files

yes i will remove css from here after merging with dev.

File: css/rules.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    left: 0;
+    width: 100%;
+    background-color: var(--primary-color);
+    color: var(--secondary-color);
+}
+
+/* HOMEPAGE STYLES */
+
+.main-heading{
+    font-family: 'Akaya Telivigala', cursive;
```
Comment (raieegulhane) [inline css/rules.css:27]: add variable for font family

**Review by raieegulhane — COMMENTED** [review]
preview link is not working.
code looks good to merge

File: css/rules.css (hunk 1 above)
Comment (muskan-gupta-0803) [reply css/rules.css:27]: okay , will add

**Review by shivamSspirit — APPROVED** [review]
good to merge

**Review by kr-sarvesh — COMMENTED** [review]
Everything looking okay to me

**Pull request merged**